meta-facebook/yv3-vf/src/platform/plat_hwmon.c

```diff
@@ -121,4 +121,22 @@ void adc_upper_bound_polling(struct k_work *work)
 		sys_write32(0x000000FF, ADC_INTERRUPT_STATUS_REG);
 
 	k_work_schedule((struct k_work_delayable *)work, K_SECONDS(1));
+}
+
+void set_exp_pwrgd_pin(void)
+{
+	uint8_t val;
+
+	switch (get_e1s_hsc_config()) {
+	case CONFIG_HSC_ADM1278:
+	case CONFIG_HSC_MAXIN:
+	case CONFIG_HSC_MPS:
+		val = gpio_get(PWRGD_P12V_AUX);
```

Comment:
Are there breaks missing?  MPS and BYPASS both set `val` but to different GPIOs.  The code on line 134 ends up being a no-op.

Reply:
I resubmitted the PR and modified the break and rebase issues
Thank you for your reminder

Reply:
@HungYi-Li - It looks like this still needs a rebase.  I don't know if you missed doing a fetch before the rebase or there has just been other changes to these files in the meantime.

Reply:
Hi @williamspatrick ,
I moved this PR to #1139 , closed this.

Reply:
Hi @williamspatrick ,
I resubmitted the PR, thanks.